Formula/arxiv_latex_cleaner.rb

```diff
@@ -0,0 +1,39 @@
+class ArxivLatexCleaner < Formula
+  include Language::Python::Virtualenv
+
+  desc "Clean LaTeX code to submit to arXiv"
+  homepage "https://github.com/google-research/arxiv-latex-cleaner"
+  url "https://files.pythonhosted.org/packages/d7/82/b32e1991ee2a8fd0db2f02ba8e135430aaeed9507e8008d5e9c8beec8eff/arxiv_latex_cleaner-0.1.27.tar.gz"
+  sha256 "f347fc6082417316247dca97bb76f15ac677eb1f5e65f091013086a30cac4eda"
+  license "Apache-2.0"
+  head "https://github.com/google-research/arxiv-latex-cleaner.git", branch: "main"
+
+  depends_on "pillow"
+  depends_on "python@3.9"
```

Comment:
Can we use `python@3.10`?

```suggestion
  depends_on "python@3.10"
```

Reply:
I had started with this, but adding `depends_on "pillow"` appeared to break. Rolling back to prior version resolves dependency issue. If there's a better way around this, would be great to know.

Reply:
Ah, I see. We have `pillow` -> `libraqm` ->  `harfbuzz` -> `glib` -> `python@3.9` dependency chain for runtime. And it breaks the idea of having `pillow` built with multiple python versions.

But for the current formula, it looks safe to add it to the `audit_exceptions/versioned_dependencies_conflicts_allowlist.json` list and switch to `python@3.10`. What do you think @carlocab?

Reply:
Fine with that, though all the commits will need to be squashed because @BrewTestBot can't squash commits that modify non-formula files.